pages/index.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <main>
-    <div class="prose desktop-gutters px-4 sm:px-6">
-      <div class="my-4">
+    <div class="desktop-gutters px-4 sm:px-6">
+      <div class="not-prose my-4">
```

Comment:
The prose classes take precedence to inline HTML classes so this class is required when there is an exception to the styling.